src/bootstrap/compile.rs

```diff
@@ -1166,14 +1166,18 @@ impl Step for Assemble {
                 let llvm_bin_dir = Path::new(llvm_bin_dir.trim());
                 builder.copy(&llvm_bin_dir.join(&src_exe), &libdir_bin.join(&dst_exe));
 
-                // Since we've already built the LLVM tools, install them to the sysroot.
-                // This is the equivalent of installing the `llvm-tools-preview` component via
-                // rustup, and lets developers use a locally built toolchain to
-                // build projects that expect llvm tools to be present in the sysroot
-                // (e.g. the `bootimage` crate).
-                for tool in LLVM_TOOLS {
-                    let tool_exe = exe(tool, target_compiler.host);
-                    builder.copy(&llvm_bin_dir.join(&tool_exe), &libdir_bin.join(&tool_exe));
+                // Not all LLVM tools are available when downloaded from CI,
+                // so don't try to copy them to the sysroot
```

Comment:
Can we opportunistically copy the ones that are available instead? I would prefer to not have entirely differing behavior with llvm-from-ci, since that seems particularly inconvenient. (If we're missing a few tools that seems better!)